eclass/git-r3.eclass

```diff
@@ -760,6 +799,29 @@ git-r3_fetch() {
 			fi
 		fi
 
+		if [[ ${EGIT_LFS} != no ]]; then
+			# Fetch the LFS files from the current ref (if any)
+			local lfs_fetch_command=( git lfs fetch "${r}" )
+			local lfs_clone_type=${EGIT_LFS_CLONE_TYPE}
+
+			if [[ ${lfs_clone_type} == mirror ]]; then
+				lfs_fetch_command+=(
+					--all
+				)
+			elif [[ ${lfs_clone_type} == shallow ]]; then
+				lfs_fetch_command+=(
+					--prune
+				)
+			fi
+
+			set -- "${lfs_fetch_command[@]}"
+			echo "${@}" >&2
+			"${@}" || die
+		elif [[ -d ${GIT_DIR}/lfs && ${EGIT_LFS_CLONE_TYPE} == shallow ]]; then
```

Comment:
BTW you don't seem to be checking for invalid value anywhere.

Reply:
I guess it is fine to check for it only when `EGIT_LFS` is defined.
(I added this in the new `case` logic)